enclave_wrangler/utils.py

```diff
@@ -71,9 +73,9 @@ def get_headers(personal=False, content_type="application/json", for_curl=False,
         headers["Content-type"] = "application/json"
 
     # set_auth_token_key(current_key)
+    # Security: space added after '$ ' to prevent possible accidental interpolation
     if for_curl:
-        # headers["authorization"] = '$' + TOKEN_KEY
-        headers["authorization"] = auth_token
+        headers["authorization"] = '$ ' + TOKEN_KEY
```

Comment:
## Only known offending case
@Sigfried Can you think of any other cases where this is getting printed?

@Sigfried What do you think about this update? It still returns a curl, but you manually have to edit the `' '` out of `'$ '` for added security.